tests/test_main.py

```diff
@@ -1425,3 +1434,90 @@ class M(BaseModel):
         a: int
 
     get_type_hints(M.__config__)
+
+
+def test_named_tuple():
```

Comment:
I know test files are a mess, but please can you move this to somewhere else. I try to keep `test_main.py` for the most vanilla cases.

Reply:
Sure thing! TBH I didn't know it was for vanilla cases. Maybe a module docstring on top of the module could help. I'll try to find a better place for these bad boys

Reply:
I created a new dedicated file `test_annotated_types`. Suggestions welcome if you don't like it